proc.c

```diff
@@ -53,6 +53,11 @@ struct proc_gtp5g_qer {
     u8      qfi;
 };
 
+struct proc_gtp5g_urr {
+    u32     id;
+    u64     seid;
+};
```

Comment:
Add more info to show here, ex: Measurement Method, Reporting Triggers, Measurement Period, Volume Threshold, Volume Quota, Measurement Information ...etc